dependencyManagement/build.gradle.kts

```diff
@@ -92,6 +92,7 @@ val DEPENDENCIES = listOf(
         "org.awaitility:awaitility:4.0.3",
         "org.codehaus.mojo:animal-sniffer-annotations:1.20",
         "org.curioswitch.curiostack:protobuf-jackson:1.2.0",
+        "org.jctools:jctools-core:3.2.0",
```

Comment:
From looking the benchmarks, I think making the general change of signaling instead of polling makes sense to me. Let's not add this dependency in this PR though, we'll need to figure out how to vendor in only the `MpscQueue` to keep the size down if we find it to be significant (it seems like we might be but let's think about it separately)

Reply:
Yeah, lets get the MpscQueue in a different PR. @anuraaga Any help would be great !